Parse/PFObject.h

```diff
@@ -623,13 +623,13 @@ NS_REQUIRES_PROPERTY_DEFINITIONS
  @param objects The list of objects to fetch.
  @param error Pointer to an `NSError` that will be set if necessary.
  */
-+ (void)fetchAll:(PF_NULLABLE NSArray *)objects error:(NSError **)error;
++ (instancetype)fetchAll:(PF_NULLABLE NSArray *)objects error:(NSError **)error;
```

Comment:
This one should have an `NSArray` return type, since the block has an `NSArray` as a result.